variants/esp32s3-spiram-octo/sdkconfig.defaults.patch

```diff
@@ -1,4 +1,10 @@
-26c26
-< CONFIG_SPIRAM_MODE_QUAD=y
----
-> CONFIG_SPIRAM_MODE_OCT=y
+--- toolchains/esp32s3/sdkconfig.defaults	2023-08-25 14:04:18.895250858 +0200
++++ toolchains/esp32s3-foo/sdkconfig.defaults	2023-08-25 14:20:21.399457870 +0200
```

Comment:
What is this foo thing? This looks a bit odd.

Reply:
The name shouldn't matter, since it will patch the file directly, but
done.